test/CSAccounting.t.sol

```diff
@@ -2870,6 +2870,184 @@ contract CSAccountingclaimRewardsUnstETHTest is
     }
 }
 
+contract CSAccountingClaimableBondTest is CSAccountingRewardsBaseTest {
+    function test_default() public override assertInvariants {
+        _operator({ ongoing: 16, withdrawn: 0 });
+        _deposit({ bond: 32 ether });
+        _rewards({ fee: 0.1 ether });
+
+        uint256 claimableBondShares = accounting.getClaimableBondShares(0);
+
+        assertEq(
+            claimableBondShares,
+            0,
+            "claimable bond shares should be zero"
+        );
+    }
+
+    function test_WithCurve() public override assertInvariants {
+        _operator({ ongoing: 16, withdrawn: 0 });
+        _deposit({ bond: 32 ether });
+        _rewards({ fee: 0.1 ether });
+        _curve(defaultCurve);
+
+        uint256 claimableBondShares = accounting.getClaimableBondShares(0);
+
+        assertApproxEqAbs(
+            claimableBondShares,
+            stETH.getSharesByPooledEth(15 ether),
+            1 wei,
+            "claimable bond shares should be equal to the curve discount"
+        );
+    }
+
+    function test_WithLocked() public override assertInvariants {
+        _operator({ ongoing: 16, withdrawn: 0 });
+        _deposit({ bond: 32 ether });
+        _rewards({ fee: 0.1 ether });
+        _lock({ id: 0, amount: 1 ether });
+
+        uint256 claimableBondShares = accounting.getClaimableBondShares(0);
+
+        assertEq(
+            claimableBondShares,
+            0,
+            "claimable bond shares should be zero"
+        );
+    }
+
+    function test_WithCurveAndLocked() public override assertInvariants {
+        _operator({ ongoing: 16, withdrawn: 0 });
+        _deposit({ bond: 32 ether });
+        _rewards({ fee: 0.1 ether });
+        _curve(defaultCurve);
+        _lock({ id: 0, amount: 1 ether });
+
+        uint256 claimableBondShares = accounting.getClaimableBondShares(0);
+
+        assertApproxEqAbs(
+            claimableBondShares,
+            stETH.getSharesByPooledEth(14 ether),
+            1 wei,
+            "claimable bond shares should be equal to the curve discount minus locked"
+        );
+    }
+
+    function test_WithOneWithdrawnValidator() public override assertInvariants {
+        _operator({ ongoing: 16, withdrawn: 1 });
+        _deposit({ bond: 32 ether });
+        _rewards({ fee: 0.1 ether });
+
+        uint256 claimableBondShares = accounting.getClaimableBondShares(0);
+
+        assertApproxEqAbs(
+            claimableBondShares,
+            stETH.getSharesByPooledEth(2 ether),
+            1 wei,
+            "claimable bond shares should be equal to a single validator bond"
+        );
+    }
+
+    function test_WithBond() public override assertInvariants {
```

Comment:
Is it the same as `test_default`?

Reply:
Yes. However, the parent contract requires implementation. The same applies to many other tests like that